extensions/grpc/runtime/src/main/java/io/quarkus/grpc/spi/ChannelBuilderCustomizer.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.grpc.spi;
+
+import java.util.Map;
+
+import io.grpc.ManagedChannelBuilder;
+import io.quarkus.grpc.runtime.config.GrpcClientConfiguration;
+import io.vertx.grpc.client.GrpcClientOptions;
+
+/**
+ * Allow for customization of Channel building (see GitHub issue #16241 for more details).
```

Comment:
I don't think we should mention GitHub issues in Javadoc. Best include everything necessary here.

Reply:
Or write some documentation :-)

Reply:
Ah, good point :-)